src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -1052,7 +1052,7 @@ public static ImmutableArray<T> CastUp<TDerived>(ImmutableArray<TDerived> items)
         [Pure]
         public ImmutableArray<TOther> CastArray<TOther>() where TOther : class
         {
-            return new ImmutableArray<TOther>((TOther[])(object)array);
+            return new ImmutableArray<TOther>((TOther[])(object)this.array);
```

Comment:
:bulb: I don't see why this needed to change. Seems better to leave it the way it was.